pip/compat.py

```diff
@@ -41,7 +46,11 @@ def console_to_str(s):
         try:
             return s.decode(sys.__stdout__.encoding)
         except UnicodeDecodeError:
-            return s.decode('utf_8')
+            if WINDOWS:
+                from ctypes import cdll
+                return s.decode("cp" + str(cdll.kernel32.GetACP()), 'replace')
```

Comment:
You are welcome, I've learned a lot from our discussion.
Thanks for your patience